inference-engine/src/mkldnn_plugin/nodes/mkldnn_embedding_bag_packed_sum_node.cpp

```diff
@@ -70,6 +64,26 @@ void MKLDNNEmbeddingBagPackedSumNode::initSupportedPrimitiveDescriptors() {
     addSupportedPrimDesc(inDataConfigurators, {{LayoutType::ncsp, inDataPrecision}}, impl_desc_type::ref_any);
 }
 
+void MKLDNNEmbeddingBagPackedSumNode::createPrimitive() {
+    if (inputShapesDefined()) {
+        if (needPrepareParams())
+            prepareParams();
+        updateLastInputDims();
+    }
+}
+
+void MKLDNNEmbeddingBagPackedSumNode::prepareParams() {
+    const auto& parentEdges = getParentEdges();
+
+    assert(INDICES_IDX < parentEdges.size() && "indexes index is bigger then edges count");
+    _batch = getStaticShape(parentEdges[INDICES_IDX])[0];
```

Comment:
use `getParentEdgesAtPort(INDICES_IDX)[0]->getMemory().getStaticDims()`
the same for all places where you use your own `getStaticShape`

Reply:
done, but no validation here